config/vsc_hortense.py

```diff
@@ -65,7 +68,7 @@ def command(self, job):
                         'arch': 'zen2',
                     },
                     'features': [
-                        'cpu',
+                        FEATURES[CPU],
```

Comment:
Wait... this should be `FEATURES['CPU']`, right? I mean, indexing the map happens with a string `'CPU'` not a python object `CPU`

Reply:
Oh, never mind, I just checked your `constants.py` and see what you did there... `CPU='CPU'`. It does _look_ very readable like this. The only concern I have is: it changes quite short object names in de global namespace. Is that wise?

Reply:
hm, not sure if it really is a problem.
if you think it is, we could make the constants longer and do away with the dictionaries, e.g.
```
FEATURE_CPU = 'cpu'
FEATURE_GPU = 'gpu'
GPU_VENDOR_NVIDIA = 'nvidia'
GPU_VENDOR_AMD = 'amd'
```

the proper way is with namespacing, but that becomes a lot less readable

Reply:
Let's keep it like you proposed originally for now, it is indeed the cleanest way. We'll find out if it comes back to haunt us some day ;-)